discover/portal/articles/100-web-experience-management/02-building-a-site/03-creating-multiple-navigation-menus/01-site-navigation.markdown

```diff
@@ -1,7 +1,6 @@
-# Default Navigation
+# Site Navigation
 
-The most straighforward way to manage your site navigation is to use the default
-navigation. With the default navigation, you create public and private pages, and organize those pages in whatever order or structure that you see fit.
+Using the primary navigation, you create public and private pages, and organize those pages in whatever order or structure that you see fit.
```

Comment:
I'm confused by the beginning of this sentence (Using the primary navigation). What does that mean? Is it referring to the pages administration?